model/src/main/kotlin/org/ostelco/prime/model/Entities.kt

```diff
@@ -297,26 +291,26 @@ enum class ProductProperties(val s: String) {
 
 enum class ProductClass {
     SIMPLE_DATA,
+    SUBSCRIPTION,
```

Comment:
Subscription is not a Product class.  Instead, it is a payment type.
(Product Class = Membership + Payment type = Subscription) => Our Membership project which we have removed.
(Product Class = Simple Data + Payment type = Subscription) => Product which will topup `x` bytes evey `y` period.
(Product Class = Data with Validity (coming soon) + Payment type = Subscription) => We will have this.

Reply:
Maybe it could be an idea to rewrite the "payment paths" in the code to only consider the `payment[type]` field and not consider the `productClass` field at all? And maybe make the `payment[type]` field mandatory?

Reply:
Don't know yet how straight forward this will be though...